dll/ntdll/ldr/ldrapi.c

```diff
@@ -100,7 +100,7 @@ NTSTATUS
 NTAPI
 LdrUnlockLoaderLock(
     _In_ ULONG Flags,
-    _In_opt_ ULONG Cookie)
+    _In_opt_ ULONG_PTR Cookie)
 {
     NTSTATUS Status = STATUS_SUCCESS;
 
```

Comment:
Need
`DPRINT("LdrUnlockLoaderLock(%x %Ix)\n", Flags, Cookie);`
too.

Reply:
why?

Reply:
ah yes, adjustment of the formatting string.